docs/policy/external-oasis-repos.md

```diff
@@ -0,0 +1,44 @@
+
+Policy for OSG Mirroring of External CVMFS repositories
+=====================================================
+
+This document provides an overview of the policies and security understanding with regards to OSG mirroring of CVMFS
+repositories of external organizations.  This is not a service-level agreement but rather a statement of responsibilities.
```

Comment:
This is a good summary of purpose. What about intended audience? In particular, do we want to have a document that can be shared with the external organizations who have CVMFS repositories?